data_tools/fsgp_timing_data/fsgp_timing_day_1.csv

```diff
@@ -0,0 +1,62 @@
+Race Day #,1,Team #,26,,,Lap Distance (Mi),3.150,Instructions: Update cells in yellow only. Green cells auto calculate. Only enter all time values in military format.,,,,,
```

Comment:
Can we move the `fsgp_timing_data` folder into a parent folder called `static_data`?